Description: Don't start bluetoothd directly from the udev rule under systemd
 systemd has better mechanisms to start services when hardware is
 hotplugged so we simply skip this rule when running under systemd.
 The bluetooth.service will be activated via bluetooth.target instead.
Author: Michael Biebl <biebl@debian.org>
Index: bluez-4.99/scripts/bluetooth.rules.in
===================================================================
--- bluez-4.99.orig/scripts/bluetooth.rules.in	2013-02-08 01:30:34.469946744 +0100
+++ bluez-4.99/scripts/bluetooth.rules.in	2013-02-08 01:34:31.192666562 +0100
@@ -1,4 +1,6 @@
 # Run helper every time a Bluetooth device appears
 # On remove actions, bluetoothd should go away by itself
+TEST=="/run/systemd/system", GOTO="bluetooth_end"
 ACTION=="add", SUBSYSTEM=="bluetooth", RUN+="/lib/udev/bluez-udev --udev"
 ACTION=="change", SUBSYSTEM=="bluetooth", RUN+="/lib/udev/bluez-udev --udev"
+LABEL="bluetooth_end"
